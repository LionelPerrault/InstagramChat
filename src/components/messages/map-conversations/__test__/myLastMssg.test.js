import React from 'react'
import { create } from 'react-test-renderer'
import MyLastMssg from '../myLastMssg'
import MockDataElement from '../../../../utils/__mocks__/mock-dataElement'

describe('MyLastMssg Component', () => {
  let dataElement

  beforeAll(() => (dataElement = MockDataElement()))

  afterEach(() => dataElement.remove())

  it('should match snapshot', () => {
    const tree = create(<MyLastMssg lastMssgBy={24} />).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('should match snapshot and return null when lastMssgBy != session ID stored in dataElement', () => {
    const tree = create(<MyLastMssg lastMssgBy={7} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
