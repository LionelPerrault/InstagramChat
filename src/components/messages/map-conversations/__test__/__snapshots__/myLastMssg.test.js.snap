// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyLastMssg Component should match snapshot 1`] = `
<span
  className="mssg_sent"
>
  <i
    className="material-icons"
  >
    done_all
  </i>
</span>
`;

exports[`MyLastMssg Component should match snapshot and return null when lastMssgBy != session ID stored in dataElement 1`] = `null`;
